website/src/roadmap/go-sdk.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: section
+title: "Go SDK Roadmap"
+permalink: /roadmap/go-sdk/
+section_menu: section-menu/roadmap.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Go SDK Roadmap
+
+This roadmap is in progress. In the meantime, here are available resources:
+
+ - JIRA: [sdk-go](https://issues.apache.org/jira/issues/?jql=project%20%3D%20BEAM%20AND%20component%20%3D%20sdk-go) / [BEAM-2083](https://issues.apache.org/jira/browse/BEAM-2083) |
+ - Contact: [Henning Rohde](mailto:herohde@google.com)
```

Comment:
WRT the Go SDK, and having discussed this with Henning previously at this point I'm the Go SDK contact, as Henning is focusing on the broader portability effort.
`[Robert Burke](mailto:rebo@google.com)`

Otherwise LGTM.